frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.unit.spec.tsx

```diff
@@ -80,26 +82,45 @@ const databaseWithEnabledActions = createMockDatabase({
 });
 
 const implicitCreateAction = createMockImplicitQueryAction({
-  id: 1,
+  id: getNextId(),
```

Comment:
What do you think about leaving the implementation of `getNextId` in this file? I'm not sure it should be abstracted yet.

I'm not aware of other tests that need this functionality. If there are a lot of other tests that want to do this then it might warrant putting it in a util.

One thing I think cause trouble with our E2E utils is that we don't plan them well enough, creating util is easy, but making util coherent with the rest is not easy.

Reply:
Actually I already wanted to have this utility few weeks ago (can't remember the PR). 

> One thing I think cause trouble with our E2E utils is that we don't plan them well enough, creating util is easy, but making util coherent with the rest is not easy.

I will keep that in mind for the future. I believe this case is justified.

Reply:
Sounds reasonable then.